src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap4/EditPageModel.cshtml

```diff
@@ -71,7 +71,7 @@ namespace @Model.NamespaceName
         }
 
         // To protect from overposting attacks, enable the specific properties you want to bind to.
-        // For more details, see https://aka.ms/RazorPagesCRUD.
+        // For more details, see https://learn.microsoft.com/aspnet/core/blazor/forms/#mitigate-overposting-attacks.
```

Comment:
Revert ... and a NIT on the language, given that the preceding remark probably doesn't constitute "details."

```suggestion
        // For more information, see https://aka.ms/RazorPagesCRUD.
```

Reply:
sg fixed